libs/martech/attributes.js

```diff
@@ -11,6 +13,13 @@ export function processTrackingLabels(text, config, charLimit) {
   return analyticsValue;
 }
 
+function getHeaderCharLimit(str) {
+  const defaultLimit = 20;
+  if (str === null || str === undefined) return defaultLimit;
+  if (str === 'off') return 0;
```

Comment:
passing in 0 as the second value will return an empty string.  So you're turning off the value, but we want to turn off the limit.
```suggestion
  if (str === 'off') return false;
```